rakelib/test_pure.rake

```diff
@@ -0,0 +1,13 @@
+require 'rake/testtask'
+
+Rake::TestTask.new(:test_pure) do |t|
+  t.libs << 'tests'
+  t.libs << 'lib'
+  t.pattern = 'tests/**/test_pure*.rb'
+  t.warning = true
+end
+
+task :test_em_pure_ruby do
+  ENV['EM_PURE_RUBY'] = 'true'
+  Rake::Task['test_pure'].execute
+end
```

Comment:
I wonder if we could get this harness to execute the entire test suite with the Ruby reactor?